### ISSUE TITLE
Ansible 2.4 full support

### ISSUE THREAD

**Issue by nervo** [manala/ansible-roles#142]
- [x] import_tasks/include_tasks
```
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 'import_tasks' for static inclusions or 'include_tasks' for dynamic inclusions. This feature will be removed in a future release. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
[DEPRECATION WARNING]: include is kept for backwards compatibility but usage is discouraged. The module documentation details page may explain more about this rationale.. This feature will be removed in a future release. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
```
- [x] meta/min_ansible_version

**Comment by chalasr** [comment]
fixed in #176

**Comment by nervo** [comment]
Php role is now ready https://github.com/manala/ansible-roles/pull/427 :)
